README.md

```diff
@@ -129,7 +125,7 @@ Each image contains the following metadata:
 - git.dirty
 - version
 
-These metadata can be seen directly on the dockerhub registry in the image layers or can be read with the following command
+These metadata can be seen directly on the AWS ECR registry in the image layers or can be read with the following command
```

Comment:
Sorry but I've just seen that this is not the case anymore, AWS ECR doesn't provide the ability to see those metadata like dockerhub. So the only way that we have now to see those metadata is to pull first the image and then enter the `image inspect` command below. I suggest to change this phrase as follow (or feel free to improve my english :-))

*Note currently it seems that we don't have permission to do `docker pull` on AWS ECR, I've poked INFRA about this, then we can quickly test those command.*

```md
These metadata can be read with the following command:

```bash
make dockerlogin
docker pull 974517877189.dkr.ecr.eu-central-1.amazonaws.com/service-kml:develop.latest

# NOTE: jq is only used for pretty printing the json output,
# you can install it with `apt install jq` or simply enter the command without it
docker image inspect --format='{{json .Config.Labels}}' 974517877189.dkr.ecr.eu-central-1.amazonaws.com/service-kml:develop.latest | jq
```
```